tfx/components/example_gen/base_example_gen_executor.py

```diff
@@ -281,6 +281,15 @@ def Do(
     """
     self._log_startup(input_dict, output_dict, exec_properties)
 
+    input_config = example_gen_pb2.Input()
+    output_config = example_gen_pb2.Output()
+    json_format.Parse(exec_properties[utils.INPUT_CONFIG_KEY], input_config)
+    json_format.Parse(exec_properties[utils.OUTPUT_CONFIG_KEY], output_config)
+
+    for output_examples_artifact in output_dict[utils.EXAMPLES_KEY]:
```

Comment:
use artifact_utils.get_single_instance (no need to have for loop, get_single_instance will assert it only contains one), same else where

Reply:
got it, i updated it for other components as well